AndroidSDKCore/src/main/java/com/leanplum/internal/LeanplumUIEditorWrapper.java

```diff
@@ -63,6 +63,7 @@ protected LeanplumUIEditorWrapper() {
           interfaceEditorSingleton = (LeanplumUIEditor) method.invoke(null);
           if (interfaceEditorSingleton != null) {
             interfaceEditorSingleton.allowInterfaceEditing(Constants.isDevelopmentModeEnabled);
+            Log.count("sdkCounterUiEditor");
```

Comment:
`sdkCountUiEditor` (no -er) ? It's shorter and in the design doc and just as accurate, IMHO